.github/workflows/check_extensions.yml

```diff
@@ -0,0 +1,39 @@
+name: Update extension list
+
+on:
+  workflow_dispatch: # Allow running on-demand
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron:  '6 6 * * *' # every sunday
+
+jobs:
+  read_extentions:
+    name: Update Extension list
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Python 3.11
```

Comment:
Don't we need to install imageio and tifffile somewhere as well?

Reply:
I'm trying out the cool script dependencies from https://packaging.python.org/en/latest/specifications/inline-script-metadata/#inline-script-metadata